src/extensions/default/AutoUpdate/main.js

```diff
@@ -204,7 +204,6 @@ define(function (require, exports, module) {
             } else {
                 // We get here if the update started but failed
                 checkInstallationStatus();
-                filesToCache = ['.logs']; //AUTOUPDATE_PRERELEASE
```

Comment:
in line 216 & 218 should we pushing in to the filesToCache array instead of replacing it with a new one ?

Reply:
Replacing with New one is fine. as in that condition Update was not initiated only so there won't be any installer logs.